File: app/Enums/TypeNum.php
<?php

namespace App\Enums;

enum TypeNum: int {
    case ROLLNO = 0;
    case PACKAGENO = 1;
    case BUNDLENO = 2;

    public function label(): string {
        return match($this) {
            self::ROLLNO => 'ROLL NO.',
            self::PACKAGENO => 'PACKAGE NO.',
            self::BUNDLENO => 'BUNDLE NO.',
        };
    }
}

File: ShippingMark/Models/TypeNum.cs
using System.ComponentModel.DataAnnotations;
namespace ShippingMark.Models
{
    public enum TypeNum
    {
        [Display(Name = "ROLL NO.")]
        [PrintText("ROLL NO.")] // <-- Thêm dòng này
        ROLLNO,

        [Display(Name = "PACKAGE NO.")]
        [PrintText("PACKAGE NO.")] // <-- Thêm dòng này
        PACKAGENO,

        [Display(Name = "BUNDLE NO.")]
        [PrintText("BUNDLE NO.")] // <-- Thêm dòng này
        BUNDLENO,
    }
   
}